run-tests.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+IOT_SCRIPT_PATH="src/iot-setup.py"
+GGL_SCRIPT_PATH="src/ggl-setup.py"
+JSON_FILE="iot_setup_data.json"
+
+# Get all test cases
+# TODO: get all tests from all test files
+get_test_functions() {
+    grep -o "test_[[:alnum:]_]*" ./src/aws-greengrass-testing-security.py | sort -u
+}
+
+setup_and_cleanup() {
+    local test_name=$1
+
+    echo "Running IoT setup..."
+    python3 $IOT_SCRIPT_PATH set_up_core_device --region=$AWS_REGION
+
+    # Read from JSON file
+    if [ -f "$JSON_FILE" ]; then
+        THING_GROUP_NAME=$(jq -r '.THING_GROUP_NAME' $JSON_FILE)
+        echo "IoT Setup completed."
+    else
+        echo "Error: Setup data file not found"
+        exit 1
+    fi
+
+    # Set up greengrass-lite
+    # TODO: delete the pause
+    read -p "Press enter to continue with Greengrass-Lite setup..."
+
+    printf "\nRunning Greengrass-Lite setup...\n"
+    python3 $GGL_SCRIPT_PATH install_greengrass_lite_from_source --id=$COMMIT_ID --region=$AWS_REGION
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Double quote to prevent globbing and word splitting.